.github/workflows/phpunit-tests.yml

```
@@ -116,7 +117,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest ]
-        php: [ '7.2', '7.3', '7.4', '8.0', '8.1', '8.2', '8.3', '8.4' ]
+        # Test highest and lowest supported version of each major.
+        php: [ '7.2', '7.4', '8.0', '8.4' ]
         db-type: [ 'mariadb' ]
         db-version: [ '10.4', '10.6', '10.11', '11.2' ]
```

```suggestion
        db-version: [ '10.6', '10.11' ]
```
11.2 was a rolling innovation release that is no longer supported. Since those releases have a short life-cycle, I lean towards saying that innovation/rolling releases are not included in test strategies for any non-supported branch.

I've also removed `10.4` here. I've opened #10166 to remove this version from `trunk` because usage across all WordPress sites has dropped below 1%.